### DISCUSSION TITLE
Does this fix photos and videos or just photos?

### DISCUSSION THREAD

**Discussion by mrkyle577** [TheLastGimbus/GooglePhotosTakeoutHelper#242] [General]
Title. Was linked here from the Synology website which stated the script only did photos but that article was last updated in Feb. 2022. Will this script be able to add the metadata back for videos now?

https://kb.synology.com/en-ro/DSM/tutorial/How_do_I_migrate_photos_from_Google_Photos

**Comment by TheLastGimbus** [comment]
Hi!

So ummm... curently, gpth doesn't "fix" anything... because as I found out, there's not much to "fix" - google doesn't strip your photos off their metadata

bigger discussion about that is here: #195 

but you can let me know (best place would be the issue above) if your photos/videos happen to have stripped data that they do have in their jsons :+1:
